client/lib/signup/step-actions/create-account.js

```diff
@@ -0,0 +1,181 @@
+import { getTracksAnonymousUserId } from '@automattic/calypso-analytics';
+import config from '@automattic/calypso-config';
+import { defer, get, isEmpty } from 'lodash';
+import { recordRegistration } from 'calypso/lib/analytics/signup';
+import { getLocaleSlug } from 'calypso/lib/i18n-utils';
+import getToSAcceptancePayload from 'calypso/lib/tos-acceptance-tracking';
+import wpcom from 'calypso/lib/wp';
+
+export function createAccount(
```

Comment:
This is 100% untouched and moved as-is. This is necessary because otherwise all the bytes of `steps-actions` would be pulled into Stepper.

Reply:
See https://calypsop2.wordpress.com/2020/02/17/consider-splitting-up-your-exports/